### PR TITLE
test coverage improvement

### PR DESCRIPTION
**Pull request by andrewhzy**
codes not covered in ds/zset/sortedset.go is for implementation purpose, it's time consuming to track, so omit it for now.

### CODE REVIEW

File: ds/set/set_test.go [hunk 1]
```diff
@@ -449,63 +454,91 @@ func TestSet_SRem(t *testing.T) {
 
 	assertions.NoError(mySet.SRem(key, []byte("a")))
 	assertions.NoError(mySet.SRem(key, []byte("b")))
+	assertions.NoError(mySet.SRem(key, []byte("c")))
+	assertions.NoError(mySet.SRem(key, make([]byte, 4, 4)))
```
Comment (xujiajun) [inline ds/set/set_test.go:458]: see reviewdog tips: 

[golangci] reported by reviewdog 🐶
S1019: should use make([]byte, 4) instead (gosimple)

Raw Output:
ds/set/set_test.go:458:50: S1019: should use make([]byte, 4) instead (gosimple)
	assertions.NoError(mySet.SRem(key, make([]byte, 4, 4)))

File: ds/list/list_test.go [hunk 2]
```diff
@@ -377,6 +377,22 @@ func TestList_LRem7(t *testing.T) {
 	num, err = list.LRem(key, math2.MinInt, []byte("b"))
 	assertions.EqualError(err, ErrMinInt.Error())
 	assertions.Equal(0, num)
+
+	num, err = list.LRem(key, 0, []byte("item_not_exists"))
```
Comment (xujiajun) [inline ds/list/list_test.go:381]: [golangci] reported by reviewdog 🐶
ineffectual assignment to err (ineffassign)

Raw Output:
ds/list/list_test.go:381:7: ineffectual assignment to err (ineffassign)
	num, err = list.LRem(key, 0, []byte("item_not_exists"))

Comment (andrewhzy) [reply ds/list/list_test.go]: thanks for your comments, have just updated.

File: ds/set/set_test.go [hunk 3]
```diff
@@ -449,63 +454,91 @@ func TestSet_SRem(t *testing.T) {
...
-			got, err := tt.set.SUnion(tt.args.key1, tt.args.key2)
-			if (err != nil) != tt.wantErr {
-				t.Errorf("SUnion() err = %v, wantErr %v", err, tt.wantErr)
-				return
-			}
-			assertions.ElementsMatchf(got, tt.want, "SUnion() got = %v, want = %v", got, tt.want)
-		})
+}
+
+func less(b1, b2 []byte) bool {
```
Comment (xujiajun) [inline ds/set/set_test.go:532]: Why do you need such a function？

**Review by xujiajun — APPROVED** [review]

**Pull request merged**